code/_onclick/hud/hud.dm

```diff
@@ -187,3 +187,5 @@
 	else
 		to_chat(usr, "<span class ='warning'>This mob type does not use a HUD.</span>")
 
+/image
+	plane = FLOAT_PLANE // I have no fucking clue why this isn't FLOAT_PLANE by default in BYOND.
```

Comment:
uh, no they... wouldn't? Images inherit their master's plane, afaik. Have you actually tested _without_ this to see if it breaks?